### PR TITLE
#November2019Release2

### PR DESCRIPTION
**Pull request by abulsayyad123**
* Support additional fields for computers, computer accessories & electrical goods(GCW-2812)
* Upgrade Twilio Client SDK as its deprecated and replaced with Twilio's new Programmable Voice SDK(GCW-1895)
* Include orders_packages in orders#show

Please review this PR.

### CODE REVIEW

File: config/deploy/staging.rb [hunk 1]
```diff
@@ -1 +1,3 @@
 server 'api-staging.goodcity.hk:59207', user: 'deployer', roles: %w{web app db}
+
+set :branch, :'GCW-1895-twilio-upgrade'
```
Comment (steveyken) [inline config/deploy/staging.rb]: @abulsayyad123 probably want to remove this now?

File: config/secrets.yml [hunk 2]
```diff
@@ -3,11 +3,13 @@ base: &BASE
...
     auth_token: <%= ENV['TWILIO_AUTH_TOKEN'] %>
-    phone_number: '<%= ENV['TWILIO_PHONE_NUMBER'] %>'
+    phone_number: <%= ENV['TWILIO_PHONE_NUMBER'] %>
     workflow_sid: <%= ENV['TWILIO_WORKFLOW_SID'] %>
     workspace_sid: <%= ENV['TWILIO_WORKSPACE_SID'] %>
     call_app_sid: <%= ENV['TWILIO_CALL_APP_SID'] %>
-    voice_number: '<%= ENV['TWILIO_VOICE_NUMBER'] %>'
+    voice_number: <%= ENV['TWILIO_VOICE_NUMBER'] %>
+    api_key: <%= ENV['TWILIO_API_KEY'] %>
+    twilio_secret: <%= ENV['TWILIO_SECRET'] %>
```
Comment (steveyken) [inline config/secrets.yml]: @abulsayyad123 @abhinav098 cc @namrataukirde remember we need to edit the `.env` file on live before deployment to add these values

File: config/deploy/staging.rb (hunk 1 above)
Comment (abulsayyad123) [reply config/deploy/staging.rb]: Removed.

File: config/secrets.yml (hunk 2 above)
Comment (abulsayyad123) [reply config/secrets.yml]: Added changes in .env file 👍 .

**Pull request merged**